Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:2.7
+
+COPY . /securitybot
+
+env PYTHONPATH $PYTHONPATH:/securitybot
+
+RUN apt-get update && \
+    apt-get install -y mysql-client && \
```

Comment:
Done, added docker-compose with DB, bot and frontend.  Agreed kubernetes deployment is out of scope here, will keep it separate.